packages/course-serializer/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@gradebook/course-serializer",
+  "version": "0.0.1",
+  "description": "Now I’m the model of a modern major general / The venerated Virginian veteran whose men are all / Lining up, to put me up on a pedestal / Writin’ letters to relatives / Embellishin’ my elegance and eloquence / But the elephant is in the room / The truth is in ya face when ya hear the British cannons go / BOOM",
```

Comment:
I'm sorry, what? 😄

Reply:
😆 That's lerna's default description, I completely forgot!